src/app/profile/edit/form/index.tsx

```diff
@@ -69,31 +77,27 @@ const EditUserForm = () => {
           placeholder="Email Address"
           error={errors.email?.message}
         />
-        <Input
-          {...register("password")}
-          icon="icon-key text-friar-gray"
-          type="password"
-          placeholder="Password"
-          error={errors.password?.message}
-        />
-        <Input
-          {...register("confirmPassword")}
-          type="password"
-          icon="icon-key text-friar-gray"
-          placeholder="Confirm password"
-          error={errors.confirmPassword?.message}
-        />
       </div>
 
-      <Button
-        type="submit"
-        disabled={isSubmitting}
-        className="mt-5 transition duration-500 
+      <div className="flex  items-center justify-center gap-5">
+        <Button
+          type="submit"
+          disabled={isSubmitting}
+          className="mt-5 transition duration-500 
```

Comment:
Almost all buttons have `transition duration-500`. I think you can move it directly to styles for button component.